packages/contracts/scripts/randomizer-deployments/shared-randomizer-deployer.ts

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import { ethers } from "hardhat";
+import { tryVerify } from "../util/verification";
+import fs from "fs";
+import path from "path";
+
+// hide nuisance logs about event overloading
+import { Logger } from "@ethersproject/logger";
+Logger.setLogLevel(Logger.levels.ERROR);
+
+// delay to avoid issues with reorgs and tx failures
+import { delay, getConfigInputs } from "../util/utils";
+import { EXTRA_DELAY_BETWEEN_TX } from "../util/constants";
+
+/**
+ * This script was created to deploy a shared randomizer contract.
+ * It is intended to document the deployment process and provide a reference
+ * for the steps required to deploy the shared randomizer contract.
+ */
+//////////////////////////////////////////////////////////////////////////////
+// CONFIG BEGINS HERE
+//////////////////////////////////////////////////////////////////////////////
+const randomizerName = "SharedRandomizerV0";
+// if the following is undefined, a new pseudorandomAtomicContract will be deployed
+// if the following is defined, the existing pseudorandomAtomicContract will be used
+let pseudorandomAtomicContractAddress = undefined;
+// the following can be undefined if pseudorandomAtomicContractAddress is defined
+const pseudorandomAtomicContractName = "PseudorandomAtomic";
+//////////////////////////////////////////////////////////////////////////////
+// CONFIG ENDS HERE
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Shouldn't these not exist at all, and be pulled in from the provided config file instead?

Reply:
omg so sorry - not quite sure how I borked this whole entire part of this 😓 
I've updated to pull config inputs from config file, and re-ran the deployment script to ensure it works, updated the deployment logs 👍 all in [81c4842](https://github.com/ArtBlocks/artblocks-contracts/pull/898/commits/81c48421d321a331599d499e8569320375741145)